HealthcareSystem/Backend/Service/SftpService/ISftpService.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Renci.SshNet;
+using Renci.SshNet.Sftp;
+
+namespace Backend.Service.SftpService
+{
+    public interface ISftpService
```

Comment:
I think the professor said while talking to the Graphical Editor team yesterday that a better name for things like these might be Communicator rather than Service? And that naming something Service implies that it's responsibility is orchestration of domain objects, which isn't really true here. I don't mind leaving the names as is right now though and refactoring it later, since we have other classes like this one which are currently named Service and when they're renamed some package refactoring should happen as well.

Reply:
I actually wrote these classes before our session with the professor, but yeah I agree they should be renamed. Perhaps have a separate folder called Communicators or something and move all classes such as this one there? I could rename it right now or I could refactor it later. I don't mind either.

Reply:
Yeah, that's what I was thinking pretty much. Maybe we could do that for these classes now if you don't mind, and change the older classes to match later?

Reply:
Sure I don't mind. Perhaps we could do it during the company meeting soon just so everyone is on the same page when it comes to naming conventions.

Reply:
Applied in 83fc2347f54ff5e0e58491584f824e21d6768c1d.